### PR TITLE
Add instructions on how to add additional devices

### PR DESCRIPTION
**Pull request by mattias-kindborg-at-work**
Deploying the application using the deploy button is easy, but adding additional devices to the application is not apparent to a reader.

Let's add instructions on how to accomplish that given an initial deployment.

### CODE REVIEW

File: telemetry-to-azure-iot-hub/README.md [hunk 1]
```diff
@@ -135,6 +135,35 @@ At this point the camera is sending a new event every 5 seconds to the Azure IoT
...
+To add additional IoT devices to your deployed application, please open [Azure Portal](https://portal.azure.com) in a web browser and navigate to the deployed IoT Hub. In the IoT Hub, select *Devices* under *Device management* in the left pane, and then click the *Add Device* toolbar button.
+
+Assuming that the new device will be named `device02`, enter the following information and then click the *Save* button.
+
+- **Device ID**: `device02`
+- **Authentication type**: `X.509 CA Signed`
+
+With the new device created, navigate to your Key Vault instance and select *Certificates* under *Settings* in the left pane. Click the `ca` certificate and download it in the same way as you previously downloaded the `device` certificate.
+
+With the CA certificate downloaded, and assumed to have the name `keyvault.pfx`, please open a terminal and run the following commands to create a new device certificate. At this point you will need the have [OpenSSL](https://www.openssl.org/) installed on your computer.
```
Comment (mattias-fjellstrom) [inline telemetry-to-azure-iot-hub/README.md:149]: Should be `need to have ...`

File: telemetry-to-azure-iot-hub/README.md [hunk 2]
```diff
@@ -135,6 +135,35 @@ At this point the camera is sending a new event every 5 seconds to the Azure IoT
...
+
+```
+$ device_identity=device02
```
Comment (mattias-fjellstrom) [inline telemetry-to-azure-iot-hub/README.md:152]: I like the `$` prefix to indicate that these commands are entered in a terminal, but I realized now that this makes it difficult to copy all commands and run them as a batch. Should we keep them or remove them?

**Review by mattias-fjellstrom — CHANGES_REQUESTED** [review]

File: telemetry-to-azure-iot-hub/README.md (hunk 1 above)
Comment (FantasticFiasco) [reply telemetry-to-azure-iot-hub/README.md:149]: Correct, thanks for finding it!

File: telemetry-to-azure-iot-hub/README.md (hunk 2 above)
Comment (FantasticFiasco) [reply telemetry-to-azure-iot-hub/README.md:152]: You're right. We should remove them. In the code examples where we used `$`, we use it to separate command from output. But where output isn't part of the code example, we should omit `$`.

**Review by mattias-fjellstrom — APPROVED** [review]
Looks good!

**Pull request merged**